sys/kern/prof.c

```diff
@@ -0,0 +1,50 @@
+#include <sys/param.h>
+#include <sys/prof.h>
+#include <sys/interrupt.h>
+#include <sys/kmem.h>
+#include <sys/gmon.h>
+#include <sys/param.h>
+#include <sys/types.h>
+#include <sys/mimiker.h>
+#include <machine/vm_param.h>
+
+gmonparam_t _gmonparam = {.state = GMON_PROF_OFF};
+
+#if KPROF
+void init_prof(void) {
+  void *profptr;
+  gmonparam_t *p = &_gmonparam;
+
+  p->lowpc = rounddown((unsigned long)__kernel_start,
+                       HISTFRACTION * sizeof(HISTFRACTION));
```

Comment:
Why `sizeof(HISTFRACTION)`? `HISTFRACTION` is defined as `2`, so it's implicitly an `int`... but why?

Reply:
My bad it should be `HISTCOUNTER`, thanks!